test/dotnet-publish.Tests/PublishDesktopTests.cs

```diff
@@ -54,7 +55,9 @@ public PublishDesktopTests()
         [InlineData("KestrelDesktop", "http://localhost:20207", "win7-x86", "libuv.dll", false)]
         public async Task DesktopApp_WithKestrel_WorksWhenPublished(string project, string url, string runtime, string libuvName, bool forceRunnable)
         {
-            var runnable = forceRunnable || string.IsNullOrEmpty(runtime) || (RuntimeEnvironmentRidExtensions.GetLegacyRestoreRuntimeIdentifier().Contains(runtime));
+            var runnable = forceRunnable ||
```

Comment:
*nit/for discusion*
When breaking up a single-line statement joined by binary operators I prefer to make the operator the first element of the new line as this makes the relationship across operands a bit easier to see. 

e.g.
````
   forceRunnable
|| string.IsNullOrEmpty(runtime)
|| ....
&& ....